style.css

```diff
@@ -323,26 +368,40 @@ hr {
   font-size: 10px;
 }
 
+.languages {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+}
+
 .programming-lang {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
+  justify-content: space-between;
   background-color: #f7f7f9;
-  margin-right: 15px;
+
+  /* margin-right: 15px; */

```

Comment:
- [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

Reply:
> - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍